utils/validateUser.js

```diff
@@ -20,24 +20,22 @@ exports.validateRegisterInput = Joi.object({
 
   email: Joi.string().email({
     minDomainSegments: 2,
-    tlds: { allow: ['com', 'net'] },
   }),
 });
 
 exports.validateUpdateInput = Joi.object({
   firstName: Joi.string()
     .alphanum()
-    .min(3)
+    .min(2)
     .max(30),
 
   lastName: Joi.string()
     .alphanum()
-    .min(3)
+    .min(2)
     .max(30),
 
   email: Joi.string().email({
     minDomainSegments: 2,
-    tlds: { allow: ['com', 'net'] },
```

Comment:
Why have you removed it?

Reply:
We can use more domains that  will not be provided